pkg/registrymirror/store.go

```diff
@@ -199,14 +200,12 @@ func (s *FileStore) updateConfigNow() error {
 func (s *FileStore) PullSources(srcImage string) ([]string, error) {
 	// FindRegistry will parse the registries config file and cache it for future usage.
 	reg, err := sysregistriesv2.FindRegistry(s.systemContext, srcImage)
-	// Using errors.Is instead of os.IsNotExist because the return from FindRegistry
-	// wraps the error in a type that os.IsNotExist ignores.
-	if err != nil && !errors.Is(err, os.ErrNotExist) {
-		return nil, errors.Wrap(err, "failed finding registry")
+	if err != nil {
+		return nil, err
 	}
 
 	if reg == nil {
-		return []string{srcImage}, nil
+		return nil, fs.ErrNotExist
```

Comment:
nit: This is technically not true, but I also won't block this if this is urgent. If not urgent, then I think we can return `nil, nil` here and deal with it on the client-side. From what I can tell, this will be handled gracefully with the client implemented as-is

Reply:
OK, changed it to nil.  Double checked the stdlib and looks like 'majority' of the use cases for ErrNotExists are fs or virtual fs related - (are a few that are not, so be it).